Infrastructure/Persistence/Redis/Sentinel.php

```diff
@@ -64,10 +67,16 @@ public function resolveMaster(): RedisConf
                 $this->logger->error(
                     "ERROR: " . $e->getMessage()
```

Comment:
We could add the retry countdown in the error message. X retries left or something like that